traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -33,20 +33,27 @@ open class AvailabilityViewController: UIViewController {
     var optionsViewController: BookingOptionsViewController?
 
     private var passes: [Pass]?
-
-    private var datePickerCellVisible = false
-    private var dateCellIndexPath: IndexPath {
+    private var availabilities: [Date: Availability] = [:]
+    
+    private var calendarCellIndexPath: IndexPath {
         return IndexPath(row: 0, section: 0)
     }
-
-    private var datePickerCellIndexPath: IndexPath {
-        return IndexPath(row: 1, section: 0)
-    }
+    
+    private var calendar: Calendar = {
+        var cal = Calendar.current
+        if let timeZone = TimeZone(secondsFromGMT: 0) {
```

Comment:
We can't handle timezones at the moment but add a comment here to indicate that this should be refactored to handle timezones in the future. //TODO: Refactor this to handle time zones.

Reply:
PR has been updated with the requested change.